seogi/src/main/java/com/example/seogi/di/util/ModuleExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.seogi.di.util
+
+import com.example.seogi.di.Module
+import kotlin.reflect.KFunction
+import kotlin.reflect.KType
+import kotlin.reflect.KVisibility
+import kotlin.reflect.full.declaredFunctions
+
+internal fun Module.findDependencyFunctionsToRemove(
```

Comment:
`toRemove` 는 없어도 될 것 같습니다~  
아래는 그냥 제 취향 내이밍? ㅋㅋㅋ

```suggestion
internal fun Module.findKFunctions(
```

Reply:
저도 네이밍을 하면서 무언가 어색하더라고요ㅎㅎ
오둥이가 예시로 들어주신 네이밍이 훨씬 낫네요!
감사합니둥